elasticdl/python/master/main.py

```diff
@@ -94,11 +94,14 @@ def main():
         args.evaluation_data_dir,
         args.prediction_data_dir,
     )
+
+    # Make sure that records_per_task is integral multiple of minibatch_size
```

Comment:
Rephrase this comment. No need to `make sure` as it is guaranteed as a multiple of minibatch_size from the calculation below.

Reply:
Good idea, will remove it.